Doc/whatsnew/3.14.rst

```diff
@@ -408,6 +408,9 @@ http
   module allow the browser to apply its default dark mode.
   (Contributed by Yorik Hansen in :gh:`123430`.)
 
+* Added support for HTTP single-part range requests on files to
+  :class:`~http.server.SimpleHTTPRequestHandler`, as specified in :rfc:`7233`.
+  (Contributed by Andy Ling in :gh:`86809`.)
```

Comment:
```suggestion
  (Contributed by Andy Ling in :gh:`86809`.)

```

Let's ensure that we have 2 blank lines (not all sections respect this but we're improving the file whenever we can)